src/build.d

```diff
@@ -938,23 +1024,24 @@ class DependencyRef
             dep.run();
         }
 
-        if (targets.isUpToDate(dep.sources, [thisBuildScript], rebuildSources))
+        if (targets && targets.isUpToDate(dep.sources, [thisBuildScript], rebuildSources))
         {
             if (dep.sources !is null)
-                log("Skipping build of %-(%s%) as it's newer than %-(%s%)", targets, sources);
+            {
+                log("Skipping build of %-(%s%) as it's newer than %-(%s%)", targets, dep.sources);
+            }
             return;
         }
-
+        // Display the execution of the dependency
+        if (msg)
```

Comment:
Now prints the message whether or not it's a commandFunction.  If the dependnecy doesn't want to print a message here, it doesn't have to set it.